packages/os/os.spec

```diff
@@ -1,5 +1,6 @@
 %global _cross_first_party 1
 %global _is_k8s_variant %(if echo %{_cross_variant} | grep -Fqw "k8s"; then echo 1; else echo 0; fi)
+%global _is_ecs_variant %(if echo %{_cross_variant} | grep -Fqw "ecs"; then echo 1; else echo 0; fi)
```

Comment:
Non-blocking but I'm slightly concerned about expanding this since users who create custom variants with names that contain any of these substrings will accidentally include packages they might not need. I wonder if we should delimit the variant tuple into fields with `awk` then checking the fields.

Reply:
Great suggestion! Maybe we should have shared macros with your idea so that we don't increase the verbosity here.